### PR TITLE
add setup.sh to download upstream packages

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+(cd $DIR;
+
+[ -e openrtm_aist ] || git clone http://github.com/tork-a/openrtm_aist-release  openrtm_aist
+[ -e openrtm_aist_python ] || git clone http://github.com/tork-a/openrtm_aist_python-release  openrtm_aist_python
+[ -e rtshell ] || git clone http://github.com/tork-a/rtshell-release rtshell
+[ -e rtsprofile ] || git clone http://github.com/tork-a/rtsprofile-release rtsprofile
+[ -e rtctree ] || git clone http://github.com/tork-a/rtctree-release rtctree
+[ -e openhrp3 ] || git clone http://github.com/fkanehiro/openhrp3 openhrp3
```
Comment (garaemon) [inline setup.sh]: Hou about --depth=1?

$ time git clone https://github.com/fkanehiro/openhrp3.git hogehoge --depth=1
Cloning into 'hogehoge'...
remote: Counting objects: 1930, done.
remote: Compressing objects: 100% (1504/1504), done.
remote: Total 1930 (delta 620), reused 1358 (delta 385)
Receiving objects: 100% (1930/1930), 27.83 MiB | 132 KiB/s, done.
Resolving deltas: 100% (620/620), done.
hub clone https://github.com/fkanehiro/openhrp3.git hogehoge --depth=1 
2.36s user 0.81s system 0% cpu 7:12.90 total

Comment (garaemon) [inline setup.sh]: without --depth=1, it takes 30 minutes to check out. Life is short.
```
$ time git clone https://github.com/fkanehiro/openhrp3.git
Cloning into 'openhrp3'...
remote: Counting objects: 18580, done.
remote: Total 18580 (delta 0), reused 0 (delta 0)
Receiving objects: 100% (18580/18580), 110.22 MiB | 166 KiB/s, done.
Resolving deltas: 100% (12195/12195), done.
hub clone https://github.com/fkanehiro/openhrp3.git  8.62s user 2.70s system 0% cpu 30:41.00 total
```

Comment (k-okada) [inline setup.sh]: I'm afraid that may have trouble when you push your work,
http://stackoverflow.com/questions/6900103/why-cant-i-push-from-a-shallow-clone
,

If you know you never push, you'd better to download from release
repository, i.e, tork-a/openhrp3-release, but that is equal to using deb

I understand downloading and compiling all openrtm/openhrp3 takes long
time, so disabling downloading source except hrpsys-base may one option.

--
◉ Kei Okada

Comment (garaemon) [inline setup.sh]: > I'm afraid that may have trouble when you push your work,

Yeah, exactly...

**Pull request closed without merging**